common/edl/poll.edl

```diff
@@ -14,9 +14,14 @@
 
 enclave {
 
-    include "openenclave/internal/syscall/sys/poll.h"
-    include "openenclave/internal/syscall/poll.h"
-    include "openenclave/internal/syscall/types.h"
+    include "openenclave/bits/edl/syscall_types.h"
+
+    struct oe_pollfd
```

Comment:
According to our naming convention guidelines, the structs ought to have an `_t` prefix.
https://github.com/openenclave/openenclave/blob/master/docs/DevelopmentGuide.md

Reply:
These are existing types. If we want to change the names I would rather do it in a different PR.